src/execution_plan/execution_plan.c

```diff
@@ -954,7 +954,6 @@ ResultSet *ExecutionPlan_Execute(ExecutionPlan *plan) {
 	/* Replace the exception-handling breakpoint set in ExecutionPlan construction with
 	 * a new breakpoint to capture run-time errors. */
 	jmp_buf *env = QueryCtx_GetExceptionHandler();
```

Comment:
Unfortunately, I tried and that doesn't work.

As mentioned at https://github.com/RedisGraph/RedisGraph/pull/613/files/e40ad8524d8b147d42a0b367b5bb375bd3cc6f2b..66e31da8e915186fae91099a66806b196d000784#r322684752 , `inline` is a compiler hint rather than a guarantee.

The call to `setjmp` must occur on the stack on which `longjmp` is invoked from; it will be invalidated if set within a function that has already returned.

> If the function in which setjmp was called returns, it is no longer possible to safely use longjmp with the corresponding jmp_buf object. This is because the stack frame is invalidated when the function returns. Calling longjmp restores the stack pointer, which—because the function returned—would point to a non-existent and potentially overwritten or corrupted stack frame.

Reply:
Let's switch to a macro, in order to ensure inlining